sale_product_approval/models/sale_order.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 Open Source Integrators
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import date
+
+from odoo import SUPERUSER_ID, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    approved_sale = fields.Boolean(
+        related="product_id.sale_ok",
+        string="Approved for Sale",
+        store=True,
+        default=True,
+    )
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    exceptions = fields.Boolean(
```

Comment:
Field name look too short prone to name collisions 
```suggestion
    exceptions_sale_approval = fields.Boolean(
```